eva_submission/ENA_submission/json_to_ENA_json.py

```diff
@@ -0,0 +1,305 @@
+        project_description = project_data.get("description", "No description provided")
+        project_centre = project_data.get("centre", "Unknown Centre")
+
+        publication_links = [
+            {"xrefLink": {"db": pub.split(":")[0], "id": pub.split(":")[1]}}
+            for pub in project_data.get("publications", [])
+            if ":" in pub
+        ]
+
+        tax_id = project_data.get("taxId", 0)
+        scientific_name = get_scientific_name_from_ensembl(str(tax_id).strip())
+
+        related_projects = []
+        for key, field_name in [
+            ("parentProject", "parentProject"),
+            ("childProjects", "childProject"),
+            ("peerProjects", "peerProject"),
+        ]:
+            values = project_data.get(key, [])
+            if isinstance(values, str):
+                related_projects.append({field_name: values})
+            elif isinstance(values, list):
+                related_projects.extend([{field_name: value} for value in values])
+
+        project_links = [self.get_link(link) for link in project_data.get("links", [])]
+
+        ena_project_obj = {
+            "accession": project_accession,
+            "alias": project_alias,
+            "title": project_title,
+            "description": project_description,
+            "centre": project_centre,
+            "publicationLinks": publication_links,
+            "sequencingProject": {},
+            "organism": {
+                "taxonId": tax_id,
+                "scientificName": scientific_name,
+            },
+            "relatedProjects": related_projects,
+            "projectLinks": project_links,
+        }
+
+        return ena_project_obj
+
+    def _create_ena_analysis_json_obj(self, ):
+        samples_per_analysis = self._samples_per_analysis(self.eva_json_data.get('sample', []))
+        files_per_analysis = self._files_per_analysis(self.eva_json_data.get('files', []))
+
+        for analysis in self.eva_json_data['analysis']:
+            samples = samples_per_analysis[analysis.get('analysisAlias')]
+            files = files_per_analysis[analysis.get('analysisAlias')]
+
+            self._add_analysis(analysis, samples, files, self.eva_json_data['project'])
+
+    def _add_analysis(self, analysis, samples, files, project):
+        def get_centre(analysis, project):
+            return analysis.get('centre') or project.get('centre')
+
+        def get_study_attr(project):
+            alias = project.get('alias', '')
+            title = project.get('title', '')
+            if re.match(r'^PRJ(EB|NA)', alias):
+                return {"accession": alias}
+            if re.match(r'^PRJ(EB|NA)', title):
+                return {"accession": title}
+            if alias:
+                return {"refname": alias}
+            if title:
+                return {"refname": title}
+            return {}
+
+        def get_samples_refs(samples):
+            # TODO: sample id ??
```

Comment:
Also not sure about this as it's not in the CLI template, maybe should use `sampleInVCF`?

Reply:
json example contains `accession` and `alias`.Nothing explicitly mentioned in the documents sent by dipayan.. will need to check further